dual_net.py

```diff
@@ -389,14 +391,18 @@ def export_model(working_dir, model_path):
 
 
 def train(working_dir, *tf_records, steps=None):
+    tf.logging.set_verbosity(tf.logging.INFO)
     if FLAGS.use_tpu:
         tpu_cluster_resolver = tf.contrib.cluster_resolver.TPUClusterResolver(
             FLAGS.tpu_name, zone=None, project=None)
+        tpu_grpc_url = tpu_cluster_resolver.get_master()
 
         config = tpu_config.RunConfig(
-            cluster=tpu_cluster_resolver,
+            master=tpu_grpc_url,
+            evaluation_master=tpu_grpc_url,
             model_dir=working_dir,
-            save_checkpoints_steps=max(600, FLAGS.iterations_per_loop),
+            save_checkpoints_steps=max(200, FLAGS.iterations_per_loop),
```

Comment:
why are you saving checkpoints so often? seems like this probably takes some amount of time.

Reply:
per https://www.tensorflow.org/api_docs/python/tf/contrib/tpu/TPUConfig

we should probably set iterations_per_loop = save_checkpoints_steps = large number
"It is recommended to be set as number of global steps for next checkpoint."

Reply:
it didn't seem to make much difference; probably because our model is (relatively) small  If we bump the batch size further (4096?) then we'll have under 1000 steps/golden chunk, which makes this seem relatively infrequent, actually.